src/appleseed.studio/mainwindow/mainwindow.cpp

```diff
@@ -181,6 +183,16 @@ void MainWindow::open_project(const QString& filepath)
     m_project_manager.load_project(filepath.toAscii().constData());
 }
 
+void MainWindow::new_project()
```

Comment:
Maybe move before `open_project()` to follow the natural progression: new, open, save, save as, etc.